src/features/common/api/api.ts

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+import camelcaseKeys from "camelcase-keys";
+
+
+const instance = axios.create({
+    baseURL: 'https://reqres.in/api'
+})
+
+
+//I will refactor(improve) link in this function, when add the pagination logic to project!
```

Comment:
Let's probably remove this comment from the codebase and just create a task in Trello? what do you think?

Reply:
This way we won't have to maintain this comment in the future and we'll have a centralized place for all our tasks and tech debt